homeassistant/components/shelly/utils.py

```diff
@@ -285,6 +285,16 @@ def get_model_name(info: dict[str, Any]) -> str:
     return cast(str, MODEL_NAMES.get(info["type"], info["type"]))
 
 
+def get_rpc_input_name(device: RpcDevice, key: str) -> str:
+    """Get input name based from the device configuration."""
+    input_config = device.config[key]
+
+    if input_name := input_config.get("name"):
+        return cast(str, input_name)
```

Comment:
We can add coverage for this line using something like this:
https://github.com/home-assistant/core/blob/2d93831cadc95472a693ef938c9b125e43ffb315/tests/components/shelly/test_utils.py#L207-L210

Reply:
Done